podcasts/Analytics/Adapters/Tracks/TracksAdapter.swift

```diff
@@ -111,12 +117,22 @@ private extension TracksAdapter {
     }
 
     func updateAuthenticationState() {
-        guard let userId = ServerSettings.userId else {
+        if let userId = ServerSettings.userId {
+            tracksService.switchToAuthenticatedUser(withUsername: nil, userID: userId, skipAliasEventCreation: false)
+        } else {
             tracksService.switchToAnonymousUser(withAnonymousID: anonymousUUID)
-            return
         }
+        reloadABTest()
+    }
+
+    func reloadExPlat() {
```

Comment:
We need to reload ExPlat after the TrackService is created, as we need to force setting up the `platform`.